src/developers/notary_server_api.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="description" content="SwaggerUI" />
+  <title>Notary Server HTTP APIs</title>
+  <link rel="stylesheet" href="https://unpkg.com/swagger-ui-dist@5.11.0/swagger-ui.css" />
+</head>
```

Comment:
```suggestion
  <style>
    /* Fast dark mode https://github.com/swagger-api/swagger-ui/issues/5327 */
    @media (prefers-color-scheme: dark) {
      body {
        background: black;
      }

      .swagger-ui {
        filter: invert(88%) hue-rotate(180deg);
      }

      .swagger-ui .microlight {
        filter: invert(100%) hue-rotate(180deg);
      }
    }
  </style>
</head>
```

Reply:
this dark mode isn't really nice though :(

<img width="922" alt="image" src="https://github.com/tlsnotary/docs-mdbook/assets/25913766/9912b1aa-3686-4670-b033-d05b7b7a29dc">

Reply:
So are we proceeding with dark mode or light mode? 👀

Reply:
> So are we proceeding with dark mode or light mode? 👀

aha we got more votes for light mode so that it is

Reply:
@waddaboo can u help to revert the dark mode changes?

Reply:
Sure thing!

Reply:
thanks again @waddaboo !